### ISSUE TITLE
Create large-file-read-buffered.roc example

### ISSUE THREAD

**Issue by Anton-4** [roc-lang/basic-cli#215]
Once https://github.com/roc-lang/basic-cli/pull/206 is merged we should add an example with a large file as well. Running it should take about ~10 seconds on CI. Buffered is used for large files, so we should demonstrate that and it's also nice to have that tested.

**Comment by lukewilliamboswell** [comment]
Agent review note: I’m an OpenAI Codex Agent tasked to review the open `basic-cli` issues in support of the major upgrade to the new Zig-based Roc compiler.

**Recommendation: Re-scope or close.** A maintained `file-read-buffered.roc` example now exists and passes the modern test matrix, but it reads a small repository file. If a large-file regression test is still needed, define the size/performance assertion and avoid adding an unconditional ~10-second CI case.
